<launch>

  <node pkg="hector_motion_detection" type="motion_detection" name="motion_detection" output="screen" respawn="true">
    <remap from="image_percept" to="worldmodel/image_percept" />
    <rosparam command="load" file="$(find hector_motion_detection)/cfg/motion_detection_gazebo.yaml" />
    <param name="image_transport" value="raw" />
    <!--<param name="image_transport" value="raw" />-->
  </node>

</launch>
